src/helpViewer/panel.ts

```diff
@@ -264,6 +265,18 @@ export class HelpPanel {
 		} else if(msg.message === 'text'){
 			// used for logging/debugging
 			console.log(`Message (text): ${String(msg.text)}`);
+		} else if(msg.message === 'codeClicked') {
+			const code = String(msg.code || '');
+			console.log(`Code clicked: ${code}`);
+			const codeClickAction = config().get<CodeClickAction>('helpPanel.clickCodeExamples');
+			if(codeClickAction === 'Disabled' || !code){
+				// pass
+			} else if(codeClickAction === 'Copy'){
+				void vscode.env.clipboard.writeText(code);
+				void vscode.window.showInformationMessage(`Copied to clipboard: \`${code}\`.`);
```

Comment:
The message seems to look too messy in many cases:

![image](https://user-images.githubusercontent.com/4662568/176815411-970c175a-e2b8-42d7-9ba5-81e8d1073de8.png)